### PR TITLE
Add ability to mark Imports complete or incomplete and CLI improvements

### PR DESCRIPTION
**Pull request by gigawhitlocks**
#### Summary

Two commits in this PR:

1 Adds the ability to mark Imports completed via the API
2 Adds the ability to unlock incomplete Imports via the API, and provides a new endpoint to fetch Imports by TranslationID as well as updates the CLI client to use this new feature:

```shell
~/g/s/g/m/awat » awat import get installation --id bhyxo1kwpfb3ib77rij1oujube
[
    {
        "ID": "49k1z5345bnaxqx91ghabqjnuo",
        "CreateAt": 1618873724156584,
        "CompleteAt": 0,
        "StartAt": 1618873737965033,
        "LockedBy": "q4jwzu1skiyrfqdu1syho8iach",
        "TranslationID": "6thxx8cdxfrpfq1gfkpduq4eda",
        "Resource": "cloud-awat-dev-ian/6thxx8cdxfrpfq1gfkpduq4eda.zip",
        "Error": "",
        "InstallationID": "bhyxo1kwpfb3ib77rij1oujube",
        "Users": 76,
        "Team": "newteam",
        "State": "import-in-progress"
    },
    {
        "ID": "c433pg4estfjfk67qmtqjrjoir",
        "CreateAt": 1618879727888784,
        "CompleteAt": 0,
        "StartAt": 1618879751669439,
        "LockedBy": "dhbnr7as6iys3m79yi3174f1no",
        "TranslationID": "c1k9gaqt33ffjrbj3cp8d47r6w",
        "Resource": "cloud-awat-dev-ian/c1k9gaqt33ffjrbj3cp8d47r6w.zip",
        "Error": "",
        "InstallationID": "bhyxo1kwpfb3ib77rij1oujube",
        "Users": 76,
        "Team": "newteam",
        "State": "import-in-progress"
    },
    {
        "ID": "k9iygwyp7f8pxf9objqfn41rme",
        "CreateAt": 1618880695473342,
        "CompleteAt": 1618880751239,
        "StartAt": 1618880696746248,
        "LockedBy": "",
        "TranslationID": "tni9x4pcqtnk9rj5me1rhbntpo",
        "Resource": "cloud-awat-dev-ian/tni9x4pcqtnk9rj5me1rhbntpo.zip",
        "Error": "",
        "InstallationID": "bhyxo1kwpfb3ib77rij1oujube",
        "Users": 76,
        "Team": "oldteam",
        "State": "import-complete"
    }
]
~/g/s/g/m/awat » awat import get import --id k9iygwyp7f8pxf9objqfn41rme 
{
    "ID": "k9iygwyp7f8pxf9objqfn41rme",
    "CreateAt": 1618880695473342,
    "CompleteAt": 1618880751239,
    "StartAt": 1618880696746248,
    "LockedBy": "",
    "TranslationID": "tni9x4pcqtnk9rj5me1rhbntpo",
    "Resource": "cloud-awat-dev-ian/tni9x4pcqtnk9rj5me1rhbntpo.zip",
    "Error": "",
    "InstallationID": "bhyxo1kwpfb3ib77rij1oujube",
    "Users": 76,
    "Team": "oldteam",
    "State": "import-complete"
}
~/g/s/g/m/awat » awat import get translation --id tni9x4pcqtnk9rj5me1rhbntpo
[
    {
        "ID": "k9iygwyp7f8pxf9objqfn41rme",
        "CreateAt": 1618880695473342,
        "CompleteAt": 1618880751239,
        "StartAt": 1618880696746248,
        "LockedBy": "",
        "TranslationID": "tni9x4pcqtnk9rj5me1rhbntpo",
        "Resource": "cloud-awat-dev-ian/tni9x4pcqtnk9rj5me1rhbntpo.zip",
        "Error": "",
        "InstallationID": "bhyxo1kwpfb3ib77rij1oujube",
        "Users": 76,
        "Team": "oldteam",
        "State": "import-complete"
    }
]
```

#### Ticket Link

No ticket, these changes support https://github.com/mattermost/mattermost-cloud/pull/439

### CODE REVIEW

**Review by Szymongib — APPROVED** [review]
LGTM

File: cmd/awat/import.go [hunk 1]
```diff
@@ -29,44 +39,71 @@ var importCmd = &cobra.Command{
...
 		server, _ := cmd.Flags().GetString(serverFlag)
-		_ = model.NewClient(server)
-
-		installation, _ := cmd.Flags().GetString(installationID)
+		translation, _ := cmd.Flags().GetString(id)
 		awat := model.NewClient(server)
-		if (installation == "" && imprt == "") || (installation != "" && imprt != "") {
-			return errors.New("one and only one of translation-id or import-id must be specified")
+		if translation == "" {
+			return errors.New("Must provide an Translation ID to search by Translation ID")
```
Comment (gabrieljackson) [inline cmd/awat/import.go:50]: nit: error message has a circular dependency ;)
```suggestion
			return errors.New("must provide a Translation ID to search by Translation ID")
```

File: cmd/awat/import.go [hunk 2]
```diff
@@ -29,44 +39,71 @@ var importCmd = &cobra.Command{
...
+		fmt.Println("No Imports found.")
+		return nil
+	},
+}
+
+var getImportByIDCmd = &cobra.Command{
+	Use:   "import",
+	Short: "Get an Import by its ID",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		imprt, _ := cmd.Flags().GetString(id)
```
Comment (gabrieljackson) [inline cmd/awat/import.go:69]: nit: trying to save a letter here? :)

**Review by gabrieljackson — APPROVED** [review]
👍

File: cmd/awat/import.go (hunk 2 above)
Comment (gigawhitlocks) [reply cmd/awat/import.go:69]: no, `import` is a reserved word @gabrieljackson

File: cmd/awat/import.go (hunk 1 above)
Comment (gigawhitlocks) [reply cmd/awat/import.go:50]: whoops lmao

File: cmd/awat/import.go (hunk 2 above)
Comment (gigawhitlocks) [reply cmd/awat/import.go:69]: I also wouldn't name a variable `func` :stuck_out_tongue:

**Pull request merged**